lib/base-service.ts

```diff
@@ -170,6 +170,22 @@ export class BaseService {
     return this.requestWrapperInstance.getHttpClient();
   }
 
+  /**
+   * Enable retries for unfulfilled requests.
+   *
+   * @param {Map<string, number>} retryConfig configuration for retries
```

Comment:
We'll need to add some info about the supported keys/fields within the map.   But really... isn't retryConfig just an object?

Reply:
This should be `RetryOptions`. I created an interface and need to update the comment here.